src/components/SharePopover/SharePopover.scss

```diff
@@ -11,11 +12,14 @@ $block: '.#{variables.$ns}share-popover';
     }
 
     &__container {
+        @include mixins.button-reset();
+        @include mixins.focusable();
```

Comment:
1. Why we need this styles (`button-reset` and `focusable`)  here?
2. I think this styles broke UI:
![share-popover-bug](https://github.com/gravity-ui/components/assets/26501073/3b94b8fc-d122-4d66-b29e-3915b41dd94f)

Reply:
1. Because `container` element is `button` tag now. Need to reset button styles therefore.
2. Fixed by disabling portal